include/dynd/parse_util.hpp

```diff
@@ -1056,8 +1056,9 @@ std::enable_if_t<is_floating_point<T>::value, T> parse(const char *begin, const
 
   // TODO: use http://www.netlib.org/fp/dtoa.c
   char *end_ptr;
-  T value = strto<T>(begin, &end_ptr);
-  if (end_ptr - begin != end - begin) {
+  const char *cp = std::string(begin, end).c_str();
```

Comment:
This will make a full copy of the string, there's got to be a way to fix this which doesn't trigger that.